src/udev/udevadm-info.c

```diff
@@ -375,8 +395,10 @@ int info_main(int argc, char *argv[], void *userdata) {
                 { "help",                    no_argument,       NULL, 'h' },
                 { "name",                    required_argument, NULL, 'n' },
                 { "path",                    required_argument, NULL, 'p' },
+                { "property",                required_argument, NULL, ARG_PROPERTY },
                 { "query",                   required_argument, NULL, 'q' },
                 { "root",                    no_argument,       NULL, 'r' },
+                { "value",                   no_argument,       NULL, ARG_VALUE },
                 { "version",                 no_argument,       NULL, 'V' },
                 { "wait-for-initialization", optional_argument, NULL, 'w' },
```

Comment:
doesn't really matter, but I think the `}` at the right should be aligned to match up again